### ISSUE TITLE
Add  class information of the images into the loss function 

### ISSUE THREAD

**Issue by kristosh** [r0nn13/conditional-dcgan-keras#6]
Is it possible to take into account as a conditional information also the actual labels of the image 2 image translation? As I understood my input image Xa from the a belongs to A domain is transformed and mapped to Xb where b belongs to B domain and also Xb belongs to X_train while the Xa belongs to y_train. Is it possible to add the actual classes of the X_a and X_b into the loss function of the discriminator?
